plugin-cgroupv1/src/cgroupv1.rs

```diff
@@ -1,34 +1,38 @@
-use crate::oar2::probe::OAR2Probe;
 use alumet::{
     measurement::{MeasurementPoint, Timestamp},
     metrics::{error::MetricCreationError, TypedMetricId},
     pipeline::elements::error::PollError,
-    plugin::{util::CounterDiffUpdate, AlumetPluginStart},
-    resources::Resource,
+    plugin::{
+        util::{CounterDiff, CounterDiffUpdate},
+        AlumetPluginStart,
+    },
+    resources::{Resource, ResourceConsumer},
     units::{PrefixedUnit, Unit},
 };
 use std::{
+    collections::HashMap,
+    fs::File,
     io::{Read, Seek},
     result::Result::Ok,
 };
 
 #[derive(Debug, Clone)]
 pub struct Metrics {
-    pub cpu_metric: TypedMetricId<u64>,
-    pub memory_metric: TypedMetricId<u64>,
+    pub cpu_time_delta: TypedMetricId<u64>,
+    pub memory_usage: TypedMetricId<u64>,
 }
 
 impl Metrics {
     pub fn new(alumet: &mut AlumetPluginStart) -> Result<Self, MetricCreationError> {
         let nsec = PrefixedUnit::nano(Unit::Second);
 
         Ok(Self {
-            cpu_metric: alumet.create_metric::<u64>(
+            cpu_time_delta: alumet.create_metric::<u64>(
                 "cpu_time_delta",
                 nsec,
                 "Total CPU time consumed by the cgroup (in nanoseconds).",
```

Comment:
```suggestion
                "Total CPU time consumed by the cgroup.",
```
As the unit is already given in the metric (it's what was done on cgroupv2)